### ISSUE TITLE
3.4.9 Bold and Italics - limit usage to one or the other?

### ISSUE THREAD

**Issue by eliseaas** [nlbdev/nordic-accessible-epub-guidelines#71]
Would it be beneficial to specify that one should use one or the other of _em_ and _strong_? Suppliers often use both at the same time, causing issues in Braille production (at least at NB). This is a recurring issue in sidebars and figure captions; `<strong><em>Se på fuglen!</em></strong>`.

**Comment by oscarlcarlsson** [comment]
There might be cases where both are needed to distinguish things visually. But they shouldn't be used unnecessarilly.

**Comment by AndersEkl** [comment]
I read through the section and I think we should re-think this. We say that only `<em>` and `<strong>` should be used and then refer to KB for markup principles. But in the KB section they also use `<b>` and `<i>`. Since we only use `<em>` and `<strong>` we frequently use those elements incorrectly.

**Comment by oscarlcarlsson** [comment]
Interesting. Should we have a look at using <b> and <i> instead/as well?

**Comment by jonaslil** [comment]
I agree that `<i>` and `<b>` would be more suitable in many cases. And for completeness, there's also `<cite>,` which is displayed with italics and "is used to mark up the title of a cited creative work" (MDM web docs) The KB doen't mention it, however.

Because of their more vague semantics, I think `<i>` and `<b>` would be worth considering as default markup for italics and bold. Perhaps opening up for using other elements if the semantic meaning is clear and matches a specific tag.

I would, however, be very reluctant to introduce it as a *requirement* to make these semantic distinctions. My reasons for this:

- It would increase manual markup work by introducing something that's impossible to automate. Automating production is something the IT group is looking into, so we should carefully consider requirements that make this more difficult.
- The semantics of elements like `<i>`, `<strong>` and `<b>` are not very clear, making it hard to apply them consistently.
- There is no screenreader support for these elements so the user experience would not be improved.

**Comment by AndersEkl** [comment]
We hold this issue for now. Screen readers don't distinguish between different ways to achieve bold or italics. If you need to to limit the use of `<strong>` and `<em>` use editing instructions. We may come back to this in the future.
